tests/vm_test.go

```diff
@@ -1614,16 +1615,15 @@ status:
 				_, _, err := clientcmd.RunCommand(testsuite.GetTestNamespace(nil), k8sClient, "create", "-f", vmJson)
 				Expect(err).ToNot(HaveOccurred())
 
-				By("Invoking virtctl start with dry-run option")
-				startCommand := clientcmd.NewRepeatableVirtctlCommand(virtctl.COMMAND_START, "--namespace", vm.Namespace, "--dry-run", vm.Name)
-				Expect(startCommand()).To(Succeed())
+				By("Performing dry run start")
```

Comment:
`[test_id:7530]in start command` is ambiguous at best. Do we want to rename?

Reply:
Done